### PR TITLE
aws-s3-cp: add page

### PR DESCRIPTION
**Pull request by shermanhui**
This PR addresses #8406 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Screen Shot 2022-10-20 at 20 02 04

### CODE REVIEW

File: pages/common/aws-s3-cp.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aws s3 cp
+
+> Copy a local file or S3 object to another location locally or in S3.
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:3]:
```suggestion
> Copy local files or S3 objects to another location locally or in S3.
```

File: pages/common/aws-s3-cp.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/cp.html>.
+
+- Copy a local file to S3:
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:6]:
```suggestion
- Copy a file from local to a specific bucket:
```

File: pages/common/aws-s3-cp.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws s3 cp {{file_name}} s3://{{example_bucket_name}}/{{file_name}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:8]:
```suggestion
`aws s3 cp {{path/to/file}} s3://{{bucket_name}}/{{path/to/remote_file}}`
```

File: pages/common/aws-s3-cp.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Copy a file from S3 to S3:
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:10]:
```suggestion
- Copy a specific S3 object into another bucket:
```

File: pages/common/aws-s3-cp.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws s3 cp s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name}}/{{file_name_2}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:12]:
```suggestion
`aws s3 cp s3://{{bucket_name1}}/{{path/to/file}} s3://{{bucket_name2}}/{{path/to/target}}`
```

File: pages/common/aws-s3-cp.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Copy an S3 object from one bucket to another:
+
+`aws s3 cp s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name_2}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:16]:
```suggestion
`aws s3 cp s3://{{bucket_name1}}/{{path/to/file}} s3://{{bucket_name2}}`
```

File: pages/common/aws-s3-cp.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Copy a local file to S3:
+
+`aws s3 cp {{file_name}} s3://{{example_bucket_name}}/{{file_name}}.txt`
+
+- Copy a file from S3 to S3:
+
+`aws s3 cp s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name}}/{{file_name_2}}.txt`
+
+- Copy an S3 object from one bucket to another:
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:14]:
```suggestion
- Copy a specific S3 object into another bucket keeping the original name:
```

File: pages/common/aws-s3-cp.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws s3 cp s3://{{example_bucket_name}}/{{file_name}}.txt s3://{{example_bucket_name_2}}`
+
+- Recursively copy S3 objects to a local directory:
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:18]: We prefer start with a verb
```suggestion
- Copy S3 objects to a local directory recursively:
```

File: pages/common/aws-s3-cp.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws s3 cp s3://{{example_bucket_name}} . --recursive`
```
Comment (navarroaxel) [inline pages/common/aws-s3-cp.md:20]:
```suggestion
`aws s3 cp s3://{{bucket_name}} . --recursive`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**